server/services/cacheService.ts

```diff
@@ -0,0 +1,68 @@
+import type { DecisionEvidenceForm } from 'forms'
+import { createRedisClient, RedisClient } from '../data/redisClient'
+
+const EVIDENCE_TEXT_ID = 'evidenceText'
+const IS_SENSITIVE_ID = 'isSensitive'
+export default class CacheService {
+  redisClient: RedisClient
+
+  constructor() {
+    this.redisClient = createRedisClient()
+  }
+
+  async saveDecisionEvidence(
+    loggedInUser,
+    crn,
+    staffTeamCode,
+    staffCode,
+    convictionNumber,
+    decisionEvidenceForm: DecisionEvidenceForm
+  ) {
+    await this.redisClient.connect()
```

Comment:
This method and the `getDecisionEvidence` are very similar.. is there any way we can reduce the amount of duplicate code (maybe it's not worth it)?